jdaviz/configs/cubeviz/plugins/slice/slice.vue

```diff
@@ -71,6 +71,51 @@
         <span>{{ wavelength_unit }}</span>
       </v-col>
     </v-row>
+
+    <v-row class="row-no-outside-padding row-min-bottom-padding">
+      <v-col>
+        <v-tooltip top>
+          <template v-slot:activator="{ on, attrs }">
+            <v-btn color="primary" icon @click="goto_first" v-bind="attrs" v-on="on">
+              <v-icon>skip_previous</v-icon>
+            </v-btn>
+          </template>
+          <span>Jump to first</span>
+        </v-tooltip>
```

Comment:
Should each button be in its own `col` with a set number of columns so that the whole row of buttons expands to fill the width?  Or is left-justified preferred to avoid it spreading out too much for a wide sidebar?

Reply:
I would be annoying if buttons spread too far from each other. I think they should stay together but please tell me how to make them prettier.